docs/getting_started/tre_organisation.md

```diff
@@ -6,8 +6,7 @@ This [reference page](../explainers/file_structure.md) goes through the other fo
 # Your Home Folder
 Available at `/home/ivm` in your sandbox, this is your personal folder. You can use this to store any files you want to keep, but it is not backed up. If you delete a file from here, it is gone forever.
 
-If you have access to multiple sandboxes, your 
-`/home` folder will be accessible from all of them.
+<!-- If you have access to multiple sandboxes, your `/home` folder will be accessible from all of them. -->
```

Comment:
Is this actually true then? I had no idea

Reply:
I don't think that's true. I removed it.